File: tags/CSLA-3.0.1-Samples/VisualBasic/PetShop.Silverlight/Source/PetShop.UI.Silverlight.Web/IWcfSlPortal.vb
Imports System.Collections.Generic
Imports System.Linq
Imports System.Runtime.Serialization
Imports System.ServiceModel
Imports System.Text

Namespace CsSlSample.Web
    ' NOTE: If you change the interface name "IWcfSlPortal" here, you must also update the reference to "IWcfSlPortal" in Web.config.
    <ServiceContract()> _
    Public Interface IWcfSlPortal
        <OperationContract()> _
        Sub DoWork()
    End Interface
End Namespace

File: tags/CSLA-4.0-Samples/CSharp/PetShop/Source/PetShop.UI.Silverlight.Web/IWcfSlPortal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.ServiceModel;
using System.Text;

namespace CsSlSample.Web
{
    // NOTE: If you change the interface name "IWcfSlPortal" here, you must also update the reference to "IWcfSlPortal" in Web.config.
    [ServiceContract]
    public interface IWcfSlPortal
    {
        [OperationContract]
        void DoWork();
    }
}
